docs/docs/aleph_zero/sdk.md

```diff
@@ -0,0 +1,139 @@
+---
+title: SDK
+
+slug: /aleph_zero/sdk
+---
+
+SDK (Software Development Kit) can be used to interact with our contract programatically.
+
+## Build
+
+### Installation
+
+Clone repository
+
+```bash
+git clone git@github.com:invariant-labs/protocol-a0.git
+```
+
+Install NPM packages & build WASM from existing contract
+
+```bash
+cd sdk
+npm i
+npm run wasm:build
+npm run build
+```
+
+### Testing
+
+Install substrate-contracts-node
+
+```bash
+cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git
```

Comment:
I think the reader should already have installed this dependency in the Installation section.

IMO, the SDK's installation should also be included in the main Installation section. Why this way?
1. Users have to install all components related to the contract to generate ABI and WASM files, especially if they want to run build.sh or make changes within the code.
2. Avoid running the installation of some dependencies twice

Reply:
So, there can be an sdk installation section that points to the main installation and a lightweight installation, as you show above. However, there should be a description of how the reader is able to perform in both installations scenarios